### ISSUE TITLE
[IDEA] Make list-links-draggable be more like list-links... but draggable

### ISSUE THREAD

**Issue by twMat** [TiddlyWiki/TiddlyWiki5#5659]
TL:DR: *list-links-draggable* should allow for dynamic listing, not just using the static `list` field, by featuring an optional `filter` parameter. 

**Is your feature request related to a problem? Please describe.**

I wish to easily list links that are draggable where the list can reflect tiddlers from the whole, current, tiddler space.

**Describe the solution you'd like**

I'd prefer an extended implementation of *list-links-draggable* but that really does as the name implies: *list-links* but draggable - and that optionally takes *all* tiddlers into account, not necessarily just a static list. This would also harmonize with the behaviour of *list-**tagged**-draggable*. Thus:

*list-links-draggable* should should feature an optional `filter` parameter. 

If **no** `filter` parameter is used then *list-links-draggable* behaves as current. 
If the `filter` parameter is used, then the `list` field is used for a `sortby`. *Dragging* should update the `list` field to add any new items from the filtering.  For deleted items; With *list-**tagged**-draggable* deleted items stay in the `list` field. I guess whatever reason is used for that, is also applicable for *list-links-draggable*.

**Note**
I find the current *list-links-draggable* implementation a bit deceiving in practice: It is not easy to tell that one is looking at a static list in contrast to the typically "fully dynamic" lists in TW. And the very dynamic nature of drag'n droping further reinforces the impression of a "live" list.
